next.dynamic.mjs

```diff
@@ -9,6 +9,8 @@ import { serialize } from 'next-mdx-remote/serialize';
 import * as nextLocales from './next.locales.mjs';
 import * as nextConstants from './next.constants.mjs';
 import * as nextHelpers from './next.helpers.mjs';
+import rehypeAutolinkHeadings from 'rehype-autolink-headings/lib';
```

Comment:
Should the import here, really be `/lib`? just out of curiosity!

Reply:
Umm I guess VSC auto imported or I might have taken from documentation, will fix that